src/Components/Components/src/ICascadingValueSupplier.cs

```diff
@@ -13,6 +13,8 @@ internal interface ICascadingValueSupplier
 
     object? GetCurrentValue(in CascadingParameterInfo parameterInfo);
 
+    object? GetCurrentValue(object? key, in CascadingParameterInfo parameterInfo) => GetCurrentValue(parameterInfo);
```

Comment:
`ICascadingValueSupplier` is not public API, right ?

Reply:
Nope, it's internal.

Reply:
So why not just change the existing `GetCurrentValue` signature instead of adding an overload?

Reply:
I did not want to go through changing all the other providers at the time, but I can clean that up.

Reply:
Updated to avoid the default method implementation